pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageQueryThrottler.java

```diff
@@ -86,7 +89,10 @@ public void init(HelixManager helixManager) {
         .count());
 
     if (_maxServerQueryThreads > 0) {
-      _semaphore = new AdjustableSemaphore(Math.max(1, _maxServerQueryThreads * _numServers / _numBrokers), true);
+      int semaphoreLimit = Math.max(1, _maxServerQueryThreads * _numServers / _numBrokers);
+      LOGGER.info("Setting max server query threads to {} for _maxServerQueryThreads={}, {} brokers, and {} servers",
```

Comment:
(minor) We usually put colon before variables. Same for other places
```suggestion
      LOGGER.info("Setting max server query threads to: {} with maxServerQueryThreads: {}, numBrokers: {}, numServers: {}",
```